src/styles/styles.js

```diff
@@ -406,6 +406,27 @@ const styles = {
         textDecorationLine: 'none',
     },
 
+    sidebarCreateMenuItem: {
+        flexDirection: 'row',
+        borderRadius: 0,
+        paddingHorizontal: 20,
+        paddingVertical: 12,
+    },
+
+    sidebarCreateMenuIcon: {
+        width: 40,
+        height: 40,
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+
+    sidebarCreateMenuText: {
+        fontFamily: fontFamily.GTA_BOLD,
+        fontSize: variables.fontSizeLabel,
```

Comment:
Can we actually make this fontSizeNormal? So it should be 15.

Reply:
Done!